### PR TITLE
Update advanced-threat-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the Supported OS

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3606

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by Brenduns** [comment]
Reviewing some intent of the original article before proceeding. Stay tuned.

File: memdocs/intune/protect/advanced-threat-protection.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ Microsoft Defender for Endpoint works with devices that run:
 - iOS/iPadOS
 - Windows 10
 - Windows 11
+- Windows Server 2008 R2
+- Windows Server 2012 R2
+- Windows Server 2016
+- Windows Server Semi-Annual Enterprise Channel
+- Windows Server 2019 and later
+- Windows Server 2019 core edition
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/advanced-threat-protection.md:48]:
```suggestion
- Windows Server 2019 Core edition
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please merge this PR. Thanks

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**